packages/ui/src/proposals/constants/styles.ts

```diff
@@ -2,7 +2,8 @@ import styled from 'styled-components'
 
 import { ListHeaders, ListHeader } from '@/common/components/List/ListHeader'
 
-export const ProposalColLayout = '296px 148px 156px 60px 1fr'
+export const ProposalColLayout = (isCouncilMember?: boolean, isPast?: boolean) =>
+  `296px 148px 156px${isPast ? ' 60px' : ''}${isCouncilMember ? ' 80px' : ''}`
```

Comment:
O I see the `60px` were for past proposal. However there's no vote section on past proposal so this should be:
```suggestion
  `296px 148px 156px${isPast ? ' 60px' : isCouncilMember ? ' 80px' : ''}`
```